### PR TITLE
feat(crowdloan/tokenomics): nodle

### PR DESCRIPTION
**Pull request by 0xJR1**
Update Nodle to include tokenomics. 
Resolves #396

### CODE REVIEW

File: crowdloans/projects/polkadot.json [hunk 1]
```diff
@@ -31,7 +31,25 @@
...
+      "totalSupply": 8500000000,
+      "data": [
+        {
+          "name": "Mining Reserves",
+          "value": 60,
+          "color": "#45E6DC"
+        },
+        { "name": "Treasury", "value": 24, "color": "#FFB080" },
+        { "name": "Team", "value": 10, "color": "#FAD14B" },
+        { "name": "Early Investor", "value": 25, "color": "#FF8066" },
```
Comment (Helen8) [inline crowdloans/projects/polkadot.json:45]: Early investor is 4.5%

**Review by Helen8 — COMMENTED** [review]
Early Investor value should be 4.5, not 25

File: crowdloans/projects/polkadot.json (hunk 1 above)
Comment (0xJR1) [reply crowdloans/projects/polkadot.json]: updated.

File: crowdloans/rewards/polkadot.json [hunk 2]
```diff
@@ -1,4 +1,81 @@
...
+        }
+      ]
+    },
+    "rewardRules": [
+      {
+        "label": "Total Crowdloan Rewards",
+        "content": ["12,000,000 LAYR (12% of Total Supply)"]
+      },
+      {
+        "label": "Bonus",
```
Comment (Helen8) [inline crowdloans/rewards/polkadot.json:58]: label should be "Project Bonus"

File: crowdloans/rewards/polkadot.json [hunk 3]
```diff
@@ -1,4 +1,81 @@
...
+        "content": [
+          "Earlier Bird: 30% extra rewards if you contribute by 1/20/2022, 20% extra rewards after",
```
Comment (Helen8) [inline crowdloans/rewards/polkadot.json:60]: Update to:

Early Bird:
-5% bonus to those who staked in the first 24 hours, by 12/15/21 at 9PM EST
-10% bonus to stakers in the first three days of opening the crowdloan

**Review by simranparallelfi — APPROVED** [review]

File: crowdloans/rewards/polkadot.json [hunk 4]
```diff
@@ -1,4 +1,83 @@
 {
+  "2019": {
+    "cDot": "${amount * 1}",
+    "reward": {
+      "total": "5% of Total Supply",
```
Comment (sjparallelfi) [inline crowdloans/rewards/polkadot.json:5]: in the document its is 12% of total supply

Comment (simranparallelfi) [reply crowdloans/rewards/polkadot.json:5]: @Helen8 can you confirm please

File: crowdloans/rewards/polkadot.json [hunk 5]
```diff
@@ -1,4 +1,83 @@
...
+        {
+          "label": "Est. Project Rewards",
+          "formula": "${amount * 0.48}"
+        },
+        {
+          "label": "Early Bird Bonus",
+          "formula": [
+            {
+              "condition": { "rangeDate": ["12/15/21", "12/15/21"] },
+              "value": "${amount * .48 * 0.15}"
```
Comment (sjparallelfi) [inline crowdloans/rewards/polkadot.json:18]: the bonus is 5% for first 24 hrs in notion doc https://www.notion.so/parallelfinance/Batch-2-DOT-Crowdloan-345f2f01d71643febd8b3c6c935615eb

File: crowdloans/rewards/polkadot.json [hunk 6]
```diff
@@ -1,4 +1,83 @@
...
+            },
+            {
+              "condition": { "rangeDate": ["12/16/21", "12/17/21"] },
```
Comment (sjparallelfi) [inline crowdloans/rewards/polkadot.json:21]: the end date is 12/18/21

File: crowdloans/rewards/polkadot.json (hunk 4 above)
Comment (Helen8) [reply crowdloans/rewards/polkadot.json:5]: 2019 - Composable Finance - 12% of supply
2031 - Centrifuge - 15% of supply

https://www.notion.so/parallelfinance/Batch-2-DOT-Crowdloan-345f2f01d71643febd8b3c6c935615eb

File: crowdloans/rewards/polkadot.json (hunk 6 above)
Comment (Helen8) [reply crowdloans/rewards/polkadot.json:21]: -5% bonus to those who staked in the first 24 hours, ending 12/15/21 at 9PM EST
-10% bonus to stakers in the first three days of opening the crowdloan, ending 12/18/21 at 9PM EST

2019, Composable Finance

Comment (sjparallelfi) [reply crowdloans/rewards/polkadot.json:21]: i think it should be 15% bonus for the first 24 hrs that should be incentive right more bonus for early investing? @Helen8

Comment (sjparallelfi) [reply crowdloans/rewards/polkadot.json:21]: in notion it is 5%. I was also confused.

**Comment by sjparallelfi** [comment]
@0xJR1 Can u pls resolve the conflicts?

File: crowdloans/rewards/polkadot.json [hunk 7]
```diff
@@ -1,4 +1,83 @@
...
+    "rewardRules": [
+      {
+        "label": "Total Crowdloan Rewards",
+        "content": ["12,000,000 LAYR (12% of Total Supply)"]
+      },
+      {
+        "label": "Project Bonus",
+        "content": [
+          "Earlier Bird: ",
+          "- 5% bonus to those who staked in the first 24 hours, by 12/15/21 at 9PM EST",
```
Comment (sjparallelfi) [inline crowdloans/rewards/polkadot.json:61]: 5% or 15%?

File: crowdloans/rewards/polkadot.json [hunk 8]
```diff
@@ -1,4 +1,83 @@
...
+        { "text": "Early Bird: From 12/16/21 to 12/17/21", "value": "10% LAYR" }
+      ],
+      "detail": [
+        {
+          "label": "Parallel Bonus",
+          "formula": "${amount * 4}"
+        },
+        {
+          "label": "Parallel Referral Bonus",
+          "formula": "${referralAvailable ? amount * 4 * 0.05 : 0}"
```
Comment (sjparallelfi) [inline crowdloans/rewards/polkadot.json:48]: In doc, It is 10% parallel referral bonus

File: crowdloans/rewards/polkadot.json [hunk 9]
```diff
@@ -1,4 +1,83 @@
...
+        }
+      ]
+    },
+    "parallelBonus": {
+      "items": [
+        { "formula": "0.05", "logo": "/static/media/para.e1be6567.svg" },
+        { "formula": "0.15" }
+      ],
+      "tooltip": [
+        { "text": "Parallel Referral Bonus", "value": "5% PARA" },
```
Comment (sjparallelfi) [inline crowdloans/rewards/polkadot.json:37]: 10% Para

File: crowdloans/rewards/polkadot.json [hunk 10]
```diff
@@ -1,4 +1,83 @@
...
+        { "text": "Early Bird: On 12/15/21", "value": "15% LAYR" },
+        { "text": "Early Bird: From 12/16/21 to 12/17/21", "value": "10% LAYR" }
```
Comment (sjparallelfi) [inline crowdloans/rewards/polkadot.json:39]: end date => 12/18/21

**Pull request merged**